pkg/models/object.go

```diff
@@ -0,0 +1,119 @@
+package models
+
+//-----------------------------------------------------------------------------------------------------
+// NOTE: the object store is in heavy development, and the locations will likely continue to move
+//-----------------------------------------------------------------------------------------------------
+
+import "context"
+
+const (
+	StandardKindDashboard = "dashboard"
+	StandardKindPlaylist  = "playlist"
+	StandardKindFolder    = "folder"
+
+	// StandardKindDataSource: not a real kind yet, but used to define references from dashboards
+	// Types: influx, prometheus, testdata, ...
+	StandardKindDataSource = "ds"
+
+	// StandardKindPanel: currently used in two ways :( in search it defines a panel within a dashboard
+	// This is also used to refer to a panel plugin with type:  heatmap, timeseries, table, ...
+	StandardKindPanel = "panel"
```

Comment:
We don't have a standalone schema for panels that exists independent of dashboards, and we're not close to being able to express that.

Reply:
Right -- as the comment said, it is currently used a few ways 😬   and we will need to resolve that.

The model I do expect to have soon is for library panels -- it is essentially a smaller dashboard with a single panel

Reply:
After sleeping on it.. the `StandardKindPanel` makes sense if we just used it for the second scenario, ie. to identify a registered panel that's provided either by Grafana or a plugin. I wouldn't say it _it refers to a panel plugin_, because IIUC a single plugin can define multiple panels.

But, for this to work well, we would actually need to populate the object store with all the panels - is that worth doing?



> We don't have a standalone schema for panels that exists independent of dashboards, and we're not close to being able to express that.

In general, isn't a transitionary period where we have a structured entity without a thema schema something we can handle? I assume that's one of the benefits of seperate "kind registry" for object store and coremodels/entities/some-other-word